docs/UserGuide.md

```diff
@@ -341,7 +445,7 @@ You can delete all person entries from the patient book.
 
 <br>
 
-### Appointment Commands
+### **Appointment Commands**
```

Comment:
Under the `find appt` feature, can we remove lines 533 and 534, `- Use the prefix `n/` to search for names. - Use the prefix `d/` to search for dates`, since they are already in the table of prefixes?

Reply:
I think it's good to keep, because it makes it easier to skim!